ColorzCore/Program.cs

```diff
@@ -94,52 +96,71 @@ static void Main(string[] args)
                                 options.nomess = true;
                                 options.nowarn = true;
                                 break;
+
+                            case "-nocash-sym":
+                                options.nocashSym = true;
+                                break;
+
                             case "h":
                             case "-help":
                                 Console.Out.WriteLine(helpstring);
-                                return;
+                                return 0;

```

Comment:
Make a static class constant for SUCCESSFUL_VALUE and ERROR_VALUE and use textual constants instead of these 0's and 1's.

Reply:
Lines 107, 110, 116, 124, 135, 140, and 163.